src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -2335,11 +2319,11 @@ namespace HVACVariableRefrigerantFlow {
                 }
             }
 
-            VRF(VRFNum).FuelType = FuelTypeElectric;
+            VRF(VRFNum).FuelType = "Electricity";
```

Comment:
Here, it appears that FuelType is not used during the simulation, so this is OK. But check that to be sure.